boards/PDM/Inc/Io/Io_Aux1Aux2Efuse.h

```diff
@@ -0,0 +1,104 @@
+#pragma once
```

Comment:
Theres still a lot of variables names that don't match the @param in the javadoc. Can you make sure those are fixed

Reply:
Also old variable names that haven't been updated